doc/longtermstorage.md

```diff
@@ -133,7 +133,7 @@ Pravega can also use an S3-compatible storage backend such as [Dell EMC ECS](htt
     ```
     ...
     spec:
-    longtermStorage:
+      longtermStorage:
```

Comment:
Why does this line show as changed? It looks exactly the same. Please revert the change if it wasn't intended.

Reply:
This was an intentional change in indentation